tests/test_functions.py

```diff
@@ -163,10 +163,11 @@ def test_fix_suffix():
     assert corrected_path.endswith(".DATA"), f"Didn't correct {corrected_path}"
 
 
-def test_get_case_uuid(case_uuid, scratch_files):
-
+def test_get_case_uuid(case_uuid, scratch_files, monkeypatch):
     real0 = scratch_files[0]
 
+    monkeypatch.chdir(real0)
```

Comment:
Was mentioned offline, is it possible to avoid doing `chdir`?
I guess it might require some code changes on our end(?) but I think it makes it difficult to keep track of what is what.
Or do we need to `chdir` to be able to use some other packages?

Either way, don't think we necessarily have to solve this right away, but it should probably be a follow-up to try and simplify.

Reply:
We should have a think about it, but the challenge is that the code is made to run inside of an ert realization. I.e from  location `<some case>/
/realization-x/iter-y` to find what it needs.. The alternative is to derive everything from environment variables that ert sets.. That actually might be doable..